content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Schema module
+---
+
+## Overview
+
+`ytt` schemas are currently in the **experimental** phase. To use schema features, include `--enable-experiment-schema`.
+
+`ytt`'s Schema feature provides a way to verify data value YAML structures and values with the help of annotations.
+
+A schema allows you to provide default data values, and verify that each data value file provided is type checked and validated against the schema and guaranteed to be proper.
+
+---
+## Defining Schema
+Configuration Authors establish a Schema by capturing the structure in a YAML document with `#@schema/match` at the top:
+```yaml
+#@schema/match data_values=True
+---
+#! Schema contents
+```
+- `data_values` (`bool`) — whether this schema is applicable to data values. If this
+  is not set to `True`, this results in an error.
+
+Notes:
+- Schema must be defined _before_ any other files containing YAML can be processed. Therefore, a file containing a Schema document must not contain other kinds of documents.
+- files containing Schema documents will be detected among input files (i.e. those specified by `-f`).
+- the first file containing a Schema document establishes the "base". Subsequent files containing Schema documents are overlayed onto the "base" in the order they appear (identically to how Data Values files are processed).
```

Comment:
link to that description of the order in which Data Values are processed?

Reply:
I ended up rewording this to use the same verbage as data values overlays docs.